source/tutorial/create-template-app.txt

```diff
@@ -0,0 +1,112 @@
+.. _create-template-app-tutorial:
```

Comment:
is this page net-new content, or did you move from somewhere?

Reply:
also, this page doesn't seem like a tutorial to me, while it def is useful getting started content. 

perhaps you can shift it up a level to be a child page of `getting-started`?
i thought that's what we'd agreed upon